### DISCUSSION TITLE
Filter notifications by PR status

### DISCUSSION THREAD

**Discussion by jcook-uptycs** [community/community#55098] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there a way to filter the notifications page by the PR status? For example only show PRs that are open, closed, merged

https://github.com/notifications?query=reason%3Amention+is%3Aissue-or-pull-request

**Comment by hrishikesh-kocharekar-dev** [comment]
Yes, GitHub provides a way to filter notifications by pull request status. You can use the "is" keyword followed by the status you want to filter by in the search bar at the top of the notifications page.

To filter by open pull requests, you can use the search query `is:open is:pr`. This will show you all open pull requests that you have notifications for.

To filter by closed pull requests, you can use the search query `is:closed is:pr`. This will show you all closed pull requests that you have notifications for.

To filter by merged pull requests, you can use the search query `is:merged is:pr`. This will show you all merged pull requests that you have notifications for.

You can also combine these search queries with other filters, such as filtering by repository, author, or date range, to further refine your results.

Note that these filters only apply to pull requests, and not to other types of notifications such as issue comments or mentions.

**Reply by xmo-odoo** [reply]
Ignore this comment, it is completely wrong, it seems to be talking about the https://github.com/pulls?q= page which is not the subject.

**Comment by jcook-uptycs** [comment]
For https://github.com/notifications 

When using `is:pr` it shows issues and pull requests.

`is:open` `is:closed` `is:merged` has no impact on that is displayed.

**Comment by ocelotl** [comment]
Same is happening to me, if I put `is:unread` in my github notifications search box, then issues, PRs and discussions are displayed.

If I put `is:unread is:pr` discussions are no longer displayed but issues and PRs are. I get the same results when I put `is:unread is:pr is:merged is:open`, `is:unread is:pr is:merged is:closed` or `is:unread is:pr is:merged is:merged`.

This seems like an issue, does someone know how to report it?

**Comment by excalq** [comment]
It would be _really_ helpful just to filter out notifications for merged PRs. Even better if this were a setting or UI option. This would make the notifications page much more usable.

**Reply by nikhilweee** [reply]
Agree 100%

**Comment by xmo-odoo** [comment]
This would indeed be really helpful, generally notifications for closed and merged PRs are of low relevance and can be handled in bulk, the ability to filter on the PR / issue status would allow quickly sifting through those notifications, something which is currently difficult.

It's especially galling when coming back from holidays with hundreds of notifications, 80% of which are that an issue or PR was closed / merged.

**Comment by iamogbz** [comment]
Main reason I do not use this feature.

Also related previous discussion: https://github.com/orgs/community/discussions/15591

**Comment by grahamj** [comment]
I can't believe there's still no way to filter by merged.

**Comment by tiago-upbrasil** [comment]
This should be possible. I also need to filter review-requested explicitly for me, review-requested for a group. But it's not possible. The notification page is useless to me.

**Comment by chad-fossa** [comment]
Still an issue today 🥉

**Reply by herickwilke** [reply]
Yep.

**Comment by manavpatil-dev0** [comment]
Not directly in the Notifications inbox right now.

GitHub's documented notification filters support things like `reason:`, `author:`, `org:`, `repo:` and a limited set of `is:` values such as `is:issue-or-pull-request`, `is:unread`, and `is:done`. PR states like `is:open`, `is:closed`, or `is:merged` aren't listed as supported notification inbox filters. :contentReference[oaicite:0]{index=0}

Those state filters do work in the Pull Requests search/dashboard, which is probably why this feels like something the Notifications page should already support. :contentReference[oaicite:1]{index=1}

So your exact use case—"show me only notifications for open/closed/merged PRs"—looks like a missing Notifications feature rather than a query you're overlooking.

I'd definitely +1 this as a feature request. Filtering out merged or closed PR notifications would make inbox triage much faster.

If this clarified it, please mark this as the answer so others looking for PR status filters in GitHub Notifications can find it easily.
